### DISCUSSION TITLE
Would it be a good idea to create a homebrew tap?

### DISCUSSION THREAD

**Discussion by cksidharthan** [michenriksen/gokiburi#3] [Ideas]
I've tried the tool and it works awesome :) I kind of felt that it would be good if there is a homebrew tap for this tool, so that the installation will be easy.

**Comment by michenriksen** [comment]
Yes, that's a great idea! Then I guess it will also get rid of the annoying Gatekeeper/unidentified developer problem for macOS users.

I think GoReleaser has a way easily release with a Homebrew tap, I'll look into that.

Thanks!

**Reply by cksidharthan** [reply]
Awesome 😁

**Comment by michenriksen** [comment]
@cksidharthan this is now implemented. You can install the latest version (v0.1.3) using Homebrew.

Thanks again for bringing up the idea!

**Reply by cksidharthan** [reply]
Perfect! Thanks for implementing it 😉
